lesson3/hw6.py

```diff
@@ -0,0 +1,5 @@
+def int_func(x):
+    return x.title()
```

Comment:
+++

Reply:
1)
def int_f (w):
    return w.capitalize ()
2)
w_s = input ("Enter the words")
print (' '.join (map (int_f, w_s.split())))